xarray/backends/netCDF4_.py

```diff
@@ -161,10 +159,7 @@ def _netcdf4_create_group(dataset, name):
 
 
 def _nc4_require_group(ds, group, mode, create_group=_netcdf4_create_group):
-    if group in {None, "", "/"}:
-        # use the root group
-        return ds
```

Comment:
I think this was clearer before -- it's an early exit for the trivial case, rather than putting all these code in between.